foursight_core/react/api/react_routes.py

```diff
@@ -0,0 +1,279 @@
+    @route_requires_authorization
+    def reactapi_route_lambdas(environ: str):
+        request = app.current_request.to_dict()
+        return app.core.reactapi_lambdas(request=request, env=environ)
+
+    @staticmethod
+    @app.route(ROUTE_PREFIX + "reactapi/{environ}/gac/{environ_compare}", cors=CORS)
+    @route_requires_authorization
+    def reactapi_route_gac_compare(environ: str, environ_compare: str):
+        request = app.current_request.to_dict()
+        return app.core.reactapi_gac_compare(request=request, env=environ, env_compare=environ_compare)
+
+    @staticmethod
+    @app.route(ROUTE_PREFIX + "reactapi/{environ}/aws/s3/buckets", methods=["GET"], cors=CORS)
+    @route_requires_authorization
+    def reactapi_route_aws_s3_buckets(environ: str):
+        """
+        Return the list of all AWS S3 bucket names for the current AWS environment.
+        """
+        request = app.current_request.to_dict()
+        return app.core.reactapi_aws_s3_buckets(request=request, env=environ)
+
+    @staticmethod
+    @app.route(ROUTE_PREFIX + "reactapi/{environ}/aws/s3/buckets/{bucket}", methods=["GET"], cors=CORS)
+    @route_requires_authorization
+    def reactapi_route_aws_s3_buckets_keys(environ: str, bucket: str):
+        """
+        Return the list of AWS S3 bucket key names in the given bucket for the current AWS environment.
+        """
+        request = app.current_request.to_dict()
+        return app.core.reactapi_aws_s3_buckets_keys(request=request, env=environ, bucket=bucket)
+
+    @staticmethod
+    @app.route(ROUTE_PREFIX + "reactapi/{environ}/aws/s3/buckets/{bucket}/{key}", methods=["GET"], cors=CORS)
+    @route_requires_authorization
+    def reactapi_route_aws_s3_buckets_key_contents(environ: str, bucket: str, key: str):
+        """
+        Return the content of the given AWS S3 bucket key in the given bucket for the current AWS environment.
+        """
+        request = app.current_request.to_dict()
+        return app.core.reactapi_aws_s3_buckets_key_contents(request=request, env=environ, bucket=bucket, key=key)
+
+    @staticmethod
+    @app.route(ROUTE_PREFIX + "reactapi/__clearcache__", cors=CORS)
+    @route_requires_authorization
+    def reactapi_route_clear_cache(environ: str):  # Not yet implemented
+        request = app.current_request.to_dict()
+        return app.core.reactapi_clear_cache(request=request, env=environ)
+
+    @staticmethod
+    @app.route(ROUTE_PREFIX + "reactapi/{environ}/__reloadlambda__", methods=["GET"], cors=CORS)
+    @route_requires_authorization
+    def reactapi_route_reload_lambda(environ: str):
+        request = app.current_request.to_dict()
+        return app.core.reactapi_reload_lambda(request=request, env=environ, lambda_name="default")
+
+    # ----------------------------------------------------------------------------------------------
+    # Foursight React UI (static file) routes.
+    # TODO: See if there is a better way to deal with variadic paths.
```

Comment:
As we discussed the thing to do probably is serve these directly from s3, but that may need to be a feature for later.